Runtime/Scripts/DebugUtils/BillboardText.cs

```diff
@@ -6,7 +6,7 @@ public class BillboardText : MonoBehaviour
 {
     private TextMeshProUGUI _text;
     public string content = "";
-    public float yOffset = 2f;
+    public Vector3 offset = new Vector3(0f, 2f, 0f);

```

Comment:
I needed z offset to handle some object overlap/render depth issues.

x offset was also useful for some objects depending on their location on the screen